siftseek/endpoints/seeker/apply/seeker_apply_get.py

```diff
@@ -0,0 +1,32 @@
+from flask import Response, jsonify
+
+from siftseek.endpoints.seeker import seeker
+from siftseek.endpoints.helpers.query_helpers import get_job_apps_by_seeker_id_or_404
+from siftseek.models.job_application import JobApplication
+from siftseek.schemas.job_application_schema import application_schema
+
+
+@seeker.get("/apply/<int:seeker_id>")
```

Comment:
since everything refers to `applications` why not use the word application instead of `apply`? generally you want your REST paths to be related to nouns, unless you're doing something like `/<verb+noun>`

Reply:
Noted, will change this! I was just using it because it was a shorter word, but it's good to know the convention around nouns.